src/ODEmodel.jl

```diff
@@ -3,18 +3,19 @@
     number of cells in G1 or G2 phase of the cell cycle 
 """
 
-const nG1 = 6
-const nG2 = 6
+const nG1 = 8
+const nG2 = 24
 const nSp = nG1 + nG2
 
 """ Make the transition matrix. """
-function ODEjac(p::AbstractVector{T}, t::Real)::Matrix{T} where {T <: Real}
-    # p = [alpha, beta, gamma1, gamma2]
-    v1 = [-ones(nG1) * (p[3] + p[1]); -ones(nG2) * (p[4] + p[2])]
-    v2 = [ones(nG1) * p[1]; ones(nG2 - 1) * p[2]]
+function ODEjac(p::AbstractVector{T}, t::Float64) where {T <: Real}
+    # p = [alpha1, alpha2, beta1, beta2, gamma11, gamma12, gamma21, gamma22]
+    v1 = [-ones(Int(nG1/2)) * (p[5] + p[1]); -ones(Int(nG1/2)) * (p[2] + p[6]); -ones(Int(nG2/2)) * (p[3] + p[7]); -ones(Int(nG2/2)) * (p[4] + p[8])]
+    v2 = [ones(Int(nG1/2)) * p[1]; ones(Int(nG1/2)) * p[2]; ones(Int(nG2/2 - 1)) * p[3]; ones(Int(nG2/2 - 1)) * p[4]]
 
     A = diagm(0 => v1, -1 => v2)
     A[1, nSp] = 2 * p[2]
+    A[nSp, nSp-1] = p[4]
 
     lmul!(t, A)
```

Comment:
Found some documentation that StaticArrays should only be used with arrays of <100 elements, as the stress on the compiler scales with the square of the size. This replacement should work for us.

Reply:
Thank you so much.